controller/src/main/java/io/pravega/controller/store/stream/InMemoryStream.java

```diff
@@ -220,6 +224,16 @@ public String getScopeName() {
         }
 
         @Override
+        public CompletableFuture<Pair<Integer, List<Integer>>> getLatestEpoch() {
```

Comment:
I'm not sure why these are not implemented. Perhaps they are implemented as part of the other PRs for this work, but I was expecting them to be implemented here. Please clarify in any case.